src/cts/src/TreeBuilder.h

```diff
@@ -192,6 +192,7 @@ class TreeBuilder
   bool isOccupiedLoc(const Point<double>& bufferLoc);
   void commitLoc(const Point<double>& bufferLoc);
   void uncommitLoc(const Point<double>& bufferLoc);
+  void commitMoveLoc(const Point<double>& oldLoc, const Point<double>& newLoc);
```

Comment:
are commitLoc and uncommitLoc still need?

Reply:
Yes, commitLoc  is used alone in a couple of places.  uncommitLoc may be needed later.